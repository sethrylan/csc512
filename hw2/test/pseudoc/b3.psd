//source: http://www.cse.chalmers.se/edu/year/2012/course/TDA282/lect02-2x2.pdf
VAR i:INT;
BEGIN
	WHILE i>6
	DO
		i--;
	ENDWHILE;
END
