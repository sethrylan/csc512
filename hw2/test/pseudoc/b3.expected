lab1:
	iload 1		; assuming x is in variable index 1
	bipush 6
	if icmpgt lab3
	iconst 0
	goto lab4
lab3:
	iconst 1
lab4:
	ifeq lab2
	iinc 1 (-1)
	goto lab1
lab2:
